### ISSUE TITLE
Add alt text to product/coffee images for accessibility

### ISSUE THREAD

**Issue by Aguero75** [Aguero75/Next_coffee#1]
Some images in the public/app folder may be missing descriptive alt attributes, which hurts accessibility and SEO.

**Comment by Aguero75** [comment]
completed
